Package/XmlEvaulation/IVariableProvider.cs

```diff
@@ -218,31 +218,41 @@ public GitVersionExpander(string projectDir)
 
         private string ProjectDir { get; }
 
+        private string longVersion = null;
+
         private string version = null;
 
         private static TraceSource log = Log.CreateSource(nameof(GitVersionExpander));
 
         public void Expand(XElement element)
         {
-            if (!element.ToString().Contains("$(GitVersion)")) return;
-            if (version == null && string.IsNullOrWhiteSpace(ProjectDir) == false)
+            ReplaceVersion("GitVersion", 5, ref version);
+            ReplaceVersion("GitLongVersion", 4, ref longVersion);
```

Comment:
I don't really understand why we need GitLongVersion here

Reply:
Pretty sure its there because nuget packages are named with LongVersion, so in order to install/include the nuget packages in the templates we need a way to write the GitlongVersion inside a file. I thought it would be best to just add it to the system already expanding the GitVersion, instead of doing it from scratch.

This is done in `sdk/sdk.package.xml` for example